overrides/macos-override.json

```diff
@@ -33,6 +33,23 @@
             "state": "enabled",
             "minSupportedVersion": "0.22.3"
         },
+        "fingerprintingHardware": {
+            "state": "enabled",
+            "settings": {
+                "keyboard": {"type": "undefined"},
+                "hardwareConcurrency": [
+                    {"type": "number", "value": 4},
+                    {"type": "number", "value": 8, "criteria": {"arch": "AppleSilicon"}}
```

Comment:
Question: Is is a problem if we report more than actually available on a given device? I'd guess not.

Reply:
@bwaresiak I don't believe so. See deeper discussion here: https://app.asana.com/0/0/1202624285456073/f